src/tools/interop/idt/utils/log.py

```diff
@@ -0,0 +1,90 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import logging
+from typing import TextIO
+
+import config
+
+_CONFIG_LEVEL = config.log_level
+
+blue = "\x1b[34;1m"
```

Comment:
if we install python bits anyway, would it make sense to use something like https://github.com/termcolor/termcolor from python for more readable code?

Otherwise constants should be `UPPER_SNAKE_CASE`

Reply:
Thanks Andrei. Indeed termcolor is nicer. Changed to that and made sure constants are caps (+private).